projects/ontimize-web-ngx/src/lib/components/list/o-list.component.ts

```diff
@@ -373,5 +377,15 @@ export class OListComponent extends AbstractOServiceComponent<OListComponentStat
       }
     });
   }
+  get quickFilterAppearance(): MatFormFieldAppearance {
+    return this._quickFilterAppearance;
+  }
 
+  set quickFilterAppearance(value: MatFormFieldAppearance) {
+    const values = ['legacy', 'standard', 'fill', 'outline'];
+    if (values.indexOf(value) === -1) {
+      value = undefined;
```

Comment:
If 'value' is not found show warning message into console and assign default `outline` value not undefined please